public/app/features/dashboard-scene/conditional-rendering/ConditionalRenderingGroup.tsx

```diff
@@ -0,0 +1,97 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { SceneComponentProps, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+import { IconButton, useStyles2 } from '@grafana/ui';
+import { t } from 'app/core/internationalization';
+
+import { ConditionalRenderingBetween } from './ConditionalRenderingBetween';
+import { ConditionalRenderingCondition } from './ConditionalRenderingCondition';
+import { ConditionalRenderingData } from './ConditionalRenderingData';
+import { ConditionalRenderingFrom } from './ConditionalRenderingFrom';
+import { ConditionalRenderingTo } from './ConditionalRenderingTo';
+import { ConditionalRenderingVariable } from './ConditionalRenderingVariable';
+
+export interface ConditionalRenderingGroupState extends SceneObjectState {
+  value: Array<
+    | ConditionalRenderingData
+    | ConditionalRenderingFrom
+    | ConditionalRenderingTo
+    | ConditionalRenderingBetween
+    | ConditionalRenderingVariable
+    | ConditionalRenderingCondition
+    | ConditionalRenderingGroup
+  >;
+}
+
+export class ConditionalRenderingGroup extends SceneObjectBase<ConditionalRenderingGroupState> {
+  public static Component = ConditionalRenderingGroupRenderer;
+
+  public evaluate(): boolean {
```

Comment:
is there anything calling evaluate? (yet)? I

Reply:
Not yet. Today...